tests/PHPStan/Rules/Comparison/data/bug-3019.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Bug3019;
```

Comment:
Duplicate namespaces/duplicate classes

Reply:
Just use a single file used in both RuleTestCase and NodeScopeResolverTest.

Reply:
ahh good catch.

when I have some time I will work on the rule to detect duplicate namespaces :)